contrib/runners/mistral_v2/query/mistral_v2.py

```diff
@@ -102,7 +102,10 @@ def _get_workflow_tasks(self, exec_id):
         :type exec_id: ``str``
         :rtype: ``list``
         """
-        wf_tasks = tasks.TaskManager(self._client).list(workflow_execution_id=exec_id)
+        wf_tasks = [
+            tasks.TaskManager(self._client).get(task.id)
```

Comment:
This returns exactly the same dict as individual item of the old list function.  The new list function no longer returns the result attribute.